### DISCUSSION TITLE
Templeate string with custom styling aka rich text, how to?

### DISCUSSION THREAD

**Discussion by onurusluca** [nuxt-modules/i18n#2936] [Q&A]
Forgive me if this was asked before, I've searched everywhere but couldn't find.

We can currently do this with i18n:

en.json:
```
...
 "toConfirmTypeInTheBox": "To confirm, type ' {confirmText} ' in the box below",
...
```

example.vue:
```
<p class="text-sm mt-2">
  {{
     $t("profile.deleteAccount.toConfirmTypeInTheBox", {
       confirmText: user?.email,
    })
  }}
</p>
```

And we can get something like this:

> To confirm, type ' myemailatemail.com' in the box below

But what if I want to style the email? What do I do then?

---

As an example there is https://next-intl-docs.vercel.app/docs/usage/messages#rich-text
We do something like this:
```
{
  "message": "Please refer to <guidelines>the guidelines</guidelines>."
}
```

then use it like this:
```
t.rich('message', {
  guidelines: (chunks) => <a class="text-red-500" href="/guidelines">{chunks}</a>
});
```

---

There is only one way I could make it work:
```
<p v-html="$t('profile.deleteAccount.toConfirmTypeInTheBox', {
    confirmText: user?.email
    })"
    class="text-sm mt-2"
></p>
```
I don't know if this is the best way or if there are other ways. 
This way also wasn't mentioned in the docs. If there is no other way, I want to at least add this to docs, if there is I want to know and add that to docs.

Is something like this possible with this module?

**Comment by BobbieGoede** [comment]
I think you're probably looking for something like this: https://vue-i18n.intlify.dev/guide/advanced/component.html

**Reply by onurusluca** [reply]
Kind of yeah, is this possible with the Nuxt module?
